src/directives-angular/slider.ts

```diff
@@ -0,0 +1,35 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2019
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { Directive, forwardRef } from '@angular/core';
+import { NG_VALUE_ACCESSOR, NumberValueAccessor } from '@angular/forms';
+import settings from 'carbon-components/es/globals/js/settings';
+
+const prefix = settings.prefix; // eslint-disable-line prefer-destructuring
+
+const host = {
+  '(blur)': 'onTouched()',
+};
+
+// NOTE: Referring `BXSlider.eventAfterChange` seems to cause ng-packagr to package up `src/components/slider.ts` code,
+// Which is not desirable
+host[`(${prefix}-slider-changed)`] = 'onChange($event.detail.value)';
+
+@Directive({
+  selector: `${prefix}-slider[formControlName],${prefix}-slider[formControl],${prefix}-slider[ngModel]`,
+  host,
```

Comment:
If we're going to use the `host` property on the `Directive` decorator, we should probably make sure we're consistent with our usage 👍

Reply:
Talked offline with @cal-smith explaining that this inconsistency came from a workaround for a (presumable) limitation in `ngc` which doesn't seem to understand computed property names in object literal, like:

```javascript
host: {
  ...
  [`(${prefix}-slider-changed)`]: 'onChange($event.detail.value)'
  ...
}
```

Thanks @cal-smith for sharing your thoughts!